1907-05-20.xml

```diff
@@ -2376,7 +2401,552 @@
                     <p><measure type="indexNo">29242-26-11-90</measure></p>
                 </div>
             </div>
-            <div type="page" n="3"> </div>
+            <div type="page" n="3">
+                <div type="item">
```

Comment:
local and general is a section. this should be `<div type="section" element="local">`